src/csharp/Grpc.Core/Internal/UnmanagedLibrary.cs

```diff
@@ -215,5 +223,19 @@ private static class Mono
             [DllImport("__Internal")]
             internal static extern IntPtr dlsym(IntPtr handle, string symbol);
         }
+
+        /// <summary>
+        /// Similarly as for Mono on Linux, we load symbols for
+        /// dlopen and dlsym from the "libcoreclr.so",
+        /// to avoid the dependency on libc-dev Linux.
+        /// </summary>
+        private static class CoreCLR
+        {
+            [DllImport("libcoreclr.so")]
+            internal static extern IntPtr dlopen(string filename, int flags);
```

Comment:
Yes, that's right. On windows we are using LoadLibrary and on mac we are using dlopen from libSystem.dylib (all macs have that library installed).